.env

```diff
@@ -58,6 +58,7 @@ FORM_FILE_GS_UPLOAD_SIZE=100M
 GS_FILES_DIRECTORY="%kernel.project_dir%/files/gs"
 GS_ORDER_PROJECT_ID=
 GS_ORDER_ISSUETYPE_ID=
+GS_ADDRESS=
```

Comment:
I suggest moving this to `config/jira_economics.local.yaml` (with a sensible default in `jira_economics.yaml`).

Editing this as Yaml, e.g.

```yaml
  …
  gs_address: >
    GRAFISK SERVICE<br/>
    ITK Design / Dokk1<br/>
    Hack Kampmanns Plads 2, 8000 Aarhus C<br/>
    Tlf.: +45 41874601 / GrafiskService@mkb.aarhus.dk
  …
```
is much easier than 

```sh
GS_ADDRESS="GRAFISK SERVICE<br/>ITK Design / Dokk1<br/>…"
```